tools/devops/automation/templates/build/stage.yml

```diff
@@ -75,8 +75,7 @@ jobs:
     name: $(AgentPoolComputed)
     demands:
     - Agent.OS -equals Darwin
-    - Agent.OSVersion -equals 10.15
-    - macios_image -equals v1
+    - macios_image -equals v2.1     # Big Sur image with Xcode 12.4 and 12.5 installed
```

Comment:
That will fail some tests.
Only `xcode12.5` branch was tested (and fixed) for running them on BigSur